packages/logger/CHANGELOG.md

```diff
@@ -0,0 +1,38 @@
+# @vue-storefront/logger
+
+## 1.0.0-rc.5
```

Comment:
Imho no point of having those messages in here as well, before merging into main let's clean it up.